pkgs/top-level/all-packages.nix

```diff
@@ -13071,7 +13071,9 @@ with pkgs;
   tinycc = callPackage ../development/compilers/tinycc { };
 
   tinygo = callPackage ../development/compilers/tinygo {
-    inherit (llvmPackages_10) llvm clang-unwrapped lld;
+    # Use same llvm version as upstream
+    # https://github.com/tinygo-org/tinygo/blob/release/Makefile
+    inherit (llvmPackages_11) llvm clang-unwrapped lld;
```

Comment:
Currently working on upgrading to LLVM 12! And then hopefully to LLVM 13 soon after.
TinyGo indeed only supports one or two LLVM versions at a time.